docs/usage/from-a-terminal.rst

```diff
@@ -149,13 +149,17 @@ This argument allows us to set a custom commit message which is going to be used
 :code:`-d "something"` | :code:`--domain "something"`
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-    Set and test the given domain.
-
-This argument will test and give the results of the tests of the given domain.
+    Test the given domain(s). Multiple space separated DNS server can be given.
 
 .. note::
-    For this argument (and only for this argument), we are converting the given string to lowercase.
 
+    This argument take 1 or more values.
+
+    As example:
+
+    ::
+
+        $ PyFunceble -d example.org example.com
```

Comment:
$ PyFunceble -d example.org example.net


For consistency between .net|.org vs. .com